tests/conftest.py

```diff
@@ -181,10 +181,10 @@ def not_code_files_and_folders(path, names):
             return to_ignore
 
         # Ignore all compiled files and egg-info.
-        ignored = list()
-        for pattern in ["__pycache__", "*.pyc", "pip.egg-info"]:
-            ignored.extend(fnmatch.filter(names, pattern))
-        return set(ignored)
+        ignored = set()
+        for pattern in ("__pycache__", "*.pyc", "pip.egg-info"):
+            ignored.update(fnmatch.filter(names, pattern))
+        return ignored
```

Comment:
Unrelated change?

Reply:
This was caught via `check-builtin-literals`, and I took the time to improve this bit of code to be not using a list -> set, instead directly using a set directly.